### ISSUE TITLE
Websocket fails to start

### ISSUE THREAD

**Issue by binaryronin** [gibber-cc/gibberwocky#20]
System: Mac OsX 10.13.6
Live Suite: 10.0.5
Max/MSP: 8 bundled and 8.0.3

Getting lots of errors when I try to start the Gibberwocky demo project. When using the built in version of Max (that comes with Suite). I get the following: 
```
codebox: patchcord inlet out of range: deleting patchcord
ws: failed to create server: Underlying Transport Error
```

When using the standalone version of Max/Msp I get:
```
codebox: patchcord inlet out of range: deleting patchcord
ws: failed to create server: Underlying Transport Error
gen~: expr: can only set as typed-in argument
```

I'm still pretty new to Max so I don't really understand the errors.

**Comment by charlieroberts** [comment]
Yes, sorry, we're in the middle of creating a fix for this. In the meantime if you try using the `devel` branch of this repo for the Max/Ableton objects, and use `http://gibberwocky.cc/beta` for the code editor, you should have more success. You will need to choose whether you want to sync the editor to Live or Max in the editor (from the config  menu); I think it defaults to Live by default.

Fix for the master branch should be coming in the next week, and I'll update this thread accordingly.
